packages/clerk-js/src/ui/Components.tsx

```diff
@@ -94,9 +96,13 @@ export const mountComponentRenderer = (
 ): ComponentControls => {
   // TODO: Init of components should start
   // before /env and /client requests
-  const clerkRoot = document.createElement('div');
-  clerkRoot.setAttribute('id', 'clerk-components');
-  document.body.appendChild(clerkRoot);
+  let clerkRoot = document.getElementById(ROOT_ELEMENT_ID);
+
+  if (!clerkRoot) {
+    clerkRoot = document.createElement('div');
+    clerkRoot.setAttribute('id', 'clerk-components');
```

Comment:
Can we reuse that constant here?

```suggestion
    clerkRoot.setAttribute('id', ROOT_ELEMENT_ID);
```

Reply:
Fixed.